hack/lib/swagger.sh

```diff
@@ -131,9 +131,12 @@ kube::swagger::gen_api_ref_docs() {
     if [[ -e "${output_dir}/${file}" && -e "${output_tmp}/${file}" ]]; then
       echo "comparing ${output_dir}/${file} with ${output_tmp}/${file}"
 
+      # Remove the timestamp to reduce conflicts in PR(s)
+      sed -i 's/^Last updated.*$//' "${output_tmp}/${file}"
```

Comment:
If we're using sed, we should use something like https://github.com/kubernetes/kubernetes/blob/master/staging/copy.sh#L64-L71 to ensure that we don't have flag issues between macOS/Linux